### ISSUE TITLE
Developing efficient transfer learning strategies for robust scene recognition in mobile robotics using pre-trained convolutional neural networks. (arXiv:2107.11187v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#9220]
Developing efficient transfer learning strategies for robust scene recognition in mobile robotics using pre-trained convolutional neural networks. (arXiv:2107.11187v1 [cs.CV])<br>
https://ift.tt/3eVdYkd<br>
<p>We present four different robust transfer learning and data augmentation strategies for robust mobile scene recognition. By training three mobile-ready (EfficientNetB0, MobileNetV2, MobileNetV3) and two large-scale baseline (VGG16, ResNet50) convolutional neural network architectures on the widely available Event8, Scene15, Stanford40, and MIT67 datasets, we show the generalization ability of our transfer learning strategies. Furthermore, we tested the robustness of our transfer learning strategies under viewpoint and lighting changes using the KTH-Idol2 database. Also, the impact of inference optimization techniques on the general performance and the robustness under different transfer learning strategies is evaluated. Experimental results show that when employing transfer learning, Fine-Tuning in combination with extensive data augmentation improves the general accuracy and robustness in mobile scene recognition. We achieved state-of-the-art results using various baseline convolutional neural networks and showed the robustness against lighting and viewpoint changes in challenging mobile robot place recognition.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
